### PR TITLE
Allow `--flag_alias` to be used with `common` and `always` commands

### LINKED ISSUE #20081: `common` does not work with `flag_alias`
### Description of the bug:

Using the new `common` in a .bazelrc with `flag_alias` returns the following error:

```sh

ERROR: /Users/<user>/Development/cash-ios/.bazelrc: "common --flag_alias=build_config=//:build_config" disallowed. --flag_alias only supports the "build" command.
```

### Which category does this issue belong to?

CLI, Configurability

### What's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

- Create a custom `config_setting`
- Create a .bazelrc
- Attempt to add a flag alias with the `common` command

```bzl
common --flag_alias=release_variant=//:release_variant
```

### Which operating system are you running Bazel on?

macOS

### What is the output of `bazel info release`?

release 6.3.2

### If `bazel info release` returns `development version` or `(@non-git)`, tell us how you built Bazel.

_No response_

### What's the output of `git remote get-url origin; git rev-parse master; git rev-parse HEAD` ?

_No response_

### Is this a regression? If yes, please try to identify the Bazel commit where the bug was introduced.

_No response_

### Have you found anything relevant by searching the web?

_No response_

### Any other information, logs, or outputs that you want to share?

Not entirely sure if this is expected but it feels like its not given how `common` works for other flags.

### PR DESCRIPTION
**Pull request by fmeum**
Fixes #20081

### CODE REVIEW

**Comment by fmeum** [comment]
@bazel-io flag

**Comment by keertk** [comment]
@bazel-io fork 7.0.0

File: src/main/java/com/google/devtools/build/lib/runtime/BlazeOptionHandler.java [hunk 1]
```diff
@@ -578,6 +578,9 @@ private String getNotInRealWorkspaceError(Path doNotBuildFile) {
     return message;
   }
 
+  private static final Set<String> BUILD_COMMAND_ANCESTORS =
+      ImmutableSet.of("build", COMMON_PSEUDO_COMMAND, ALWAYS_PSEUDO_COMMAND);
```
Comment (katre) [inline src/main/java/com/google/devtools/build/lib/runtime/BlazeOptionHandler.java:582]: Can you also add a constant for `"build"` (and use it on line 617 also)?

**Review by katre — CHANGES_REQUESTED** [review]
LGTM, thanks for picking this up so quickly.

**Review by gregestren — APPROVED** [review]

File: src/main/java/com/google/devtools/build/lib/runtime/BlazeOptionHandler.java (hunk 1 above)
Comment (fmeum) [reply src/main/java/com/google/devtools/build/lib/runtime/BlazeOptionHandler.java]: Done

**Review by katre — APPROVED** [review]

**Comment by luispadron** [comment]
Thanks!

**Pull request closed without merging**